### PR TITLE
Add mmHg and inHg to Pressure units.

### PR DESCRIPTION
**Pull request by JonathanDavies626**
Add millimeters of mercury to pressure units.

### CODE REVIEW

**Comment by jcowgar** [comment]
I was just about to add inHg, 1 in hg = 3386.388666667 pa. I wonder if that conversion can be added to this merge w/minimal effort? I've not done any .NET programming beyond my experience yesterday, so my environment isn't quite setup yet :-/

**Comment by JonathanDavies626** [comment]
Yes, this can be added without too much effort.

File: UnitsNet/UnitDefinitions/Pressure.json [hunk 1]
```diff
@@ -283,18 +283,42 @@
...
+      "FromBaseToUnitFunc": "x*7.50061561302643e-3",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "mmHg" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "InchesOfMercury",
```
Comment (anjdreas) [inline UnitsNet/UnitDefinitions/Pressure.json]: Typo, you probably meant `InchOfMercury`

**Comment by anjdreas** [comment]
Great!

**Comment by jcowgar** [comment]
@JonathanDavies626 , sorry about causing the delay with your introduction of inHg but thank you very much for including it on this request!

**Comment by anjdreas** [comment]
Nuget 3.62 is out.

**Pull request merged**